internal/Picker/Picker.js

```diff
@@ -754,12 +780,19 @@ const PickerBase = class extends ReactComponent {
 	};
 
 	determineClasses (css, decrementerDisabled, incrementerDisabled) {
-		const {className, joined, orientation, width} = this.props;
+		const {changedBy, className, joined, orientation, width} = this.props;
 		const {pressed} = this.state;
 
+		let cssOrientation;
```

Comment:
How about adding css[changedBy] to classnames instead of this block?
And in less, we could use `.horizontal` for both cases, `:not(.arrow)` for the normal one, and `.arrow` for the new one.